client/my-sites/post-type-list/post-action-counts/index.jsx

```diff
@@ -30,6 +28,11 @@ class PostActionCounts extends PureComponent {
 		globalId: PropTypes.string,
 	};
 
+	state = {
+		likesPopoverContext: null,
+		likesPopoverPostId: null,
```

Comment:
Instead of `likesPopoverPostId`, we can have a simple boolean flag that says whether the popover is displayed or not.

Before this PR, the state was shared by the entire list and contained the `id` of displayed popover. Then the `Counts` component needed to detect whether "a popover is shown _and_ it's me".

Reply:
The `likesPopoverContext` could be a simple ref, no need to call `setState` on it.

Reply:
Thanks @jsnajdr I've addressed both comments in https://github.com/Automattic/wp-calypso/pull/45293/commits/a642f79a690900c0b51221b15f4d543559a3c847